homeassistant/components/ld2410_ble/sensor.py

```diff
@@ -74,12 +74,64 @@
     state_class=SensorStateClass.MEASUREMENT,
 )
 
+MAX_MOTION_GATES_DESCRIPTION = SensorEntityDescription(
```

Comment:
All these entities should be marked as entity category `diagnostic`. Otherwise looks good!